Sources/Core/Console.swift

```diff
@@ -63,6 +63,15 @@ extension TerminalController: TerminalControlling {
     }
 }
 
+class PrintController: TerminalControlling {
```

Comment:
Minor suggestion since it shouldn't be subclassed (and for very very minor performance gains 😄):
```suggestion
final class PrintController: TerminalControlling {
```

Reply:
If you agree I think the added factory could also be final

Reply:
Yeah that sounds good. I will add the final keyword 😄